.github/workflows/db_apply_anonimized.yml

```diff
@@ -0,0 +1,15 @@
+name: "DB - Apply Anonymized db"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  save-anonymized-db:
```

Comment:
Why did you named it 'save-anonymized-db' instead of 'apply-anonymized-db'? Name also should be 'Apply Anonymized DB'.